tox.ini

```diff
@@ -1,5 +1,17 @@
 [tox]
-envlist = py{27,34,35}-django1{8,9,10}
+envlist = py{27,34,35}-django1{8,9,10} coverage
+
+[testenv:coverage]
+basepython = python2.7
+deps =
+    Django>=1.8,<1.9
+    -Urtest/test_project/requirements_test.txt
```

Comment:
No it's not a typo : I kept the same syntax which was used in this file. I have seen this in some projects. I suppose it's for some version of tox or OS ?